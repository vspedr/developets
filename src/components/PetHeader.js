import React from 'react';
import { Icon } from 'semantic-ui-react';

export const PetHeader = () => {

  return (
    <header className="App-header">
      <div className="App-header__group">
        <span className="App-title">DeveloPets</span>
        <span className="App-header__group-intro">
          Introduce your animal buddies to your fellow developers
        </span>
      </div>
      <div className="App__icons">
        <Icon name="paw" size="huge" />
        <Icon name="paw" size='big' className="App__icons--big" />
        <Icon name="paw" size='big' />
      </div>
    </header>
  );
};
