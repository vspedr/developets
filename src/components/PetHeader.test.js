import React from 'react';
import { shallow } from 'enzyme';
import { PetHeader } from './PetHeader';

describe('A Header Component', () => {
  it('should match snapshot component', () => {
    const rendered = shallow(<PetHeader />);
    expect(rendered.html()).toMatchSnapshot();
  });
});
