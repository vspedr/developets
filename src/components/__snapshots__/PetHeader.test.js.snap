// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A Header Component should match snapshot component 1`] = `"<header class=\\"App-header\\"><div class=\\"App-header__group\\"><span class=\\"App-title\\">DeveloPets</span><span class=\\"App-header__group-intro\\">Introduce your animal buddies to your fellow developers</span></div><div class=\\"App__icons\\"><i aria-hidden=\\"true\\" class=\\"paw huge icon\\"></i><i aria-hidden=\\"true\\" class=\\"paw big icon App__icons--big\\"></i><i aria-hidden=\\"true\\" class=\\"paw big icon\\"></i></div></header>"`;
